### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by williamFalcon**
Clear up what litGPT is/does at the top of readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,8 +70,7 @@ pip install -e '.[all]'
 ---
 
 # Get started
-LitGPT is a command-line tool to use, pretrain, finetune and deploy LLMs.
-
+LitGPT is CLI and config based. Select the model and the action you want to take on that model (finetune, pretrain, deploy, etc...):
```
Comment (rasbt) [inline README.md:73]:
```suggestion
LitGPT is CLI and config-based. Select the model and the action you want to take on that model (finetune, pretrain, deploy, evaluate etc...):    
```

File: README.md [hunk 2]
```diff
@@ -25,14 +25,24 @@ Uses the latest state-of-the-art techniques:
...
-  <a href="#litgpt-design-principles">Design principles</a>
+  <a href="#training-recipes">Training recipes (YAML)</a>
 </p>
 
 </div>
 
 &nbsp;
 
+## Finetune, pretrain and deploy AI models Lightning fast ⚡⚡
+LitGPT is a command-line tool to use, pretrain, finetune and deploy LLMs. It is config based with highly-optimized recipes for training the world's largest, most powerful open-source LLMs.
```
Comment (rasbt) [inline README.md:36]:
```suggestion
LitGPT is a command-line tool to use, pretrain, finetune and deploy LLMs. It is based on configs with highly-optimized recipes for training the world's largest, most powerful open-source LLMs.     
```

**Review by rasbt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Andrei-Aksionov) [reply README.md:73]: Not particularly important, but maybe `(pretrain, finetune, evaluate, deploy, etc...)`?
Better reflects development process.

Comment (rasbt) [reply README.md:73]: Sure, happy to update the ordering from `deploy, evaluate` to `evaluate, deploy`. Haha, in reality it's probably more like `pretrain, fintune, evaluate, deploy, evaluate, finetune, evaluate, deploy ...`

Comment (Andrei-Aksionov) [reply README.md:73]: In reality it might never make it to `deploy` stage.

**Comment by williamFalcon** [comment]
@rasbt @Andrei-Aksionov can we get CI back to green please. a PR that changes a readme shouldn't fail... this shows there are real broken tests on the CI

**Comment by rasbt** [comment]
@williamFalcon Totally agree. And I had problems merging PRs because of that in recent days because I don't have the permissions to override the merges if one of the test fails for unrelated reasons. There is a proposed fix by @Andrei-Aksionov in  #1292 I believe.

**Comment by williamFalcon** [comment]
> @williamFalcon Totally agree. And I had problems merging PRs because of that in recent days because I don't have the permissions to override the merges if one of the test fails for unrelated reasons. There is a proposed fix by @Andrei-Aksionov in #1292 I believe.

awesome. let's get it unblocked for sure. i moved the discussion to #1294.

File: README.md (hunk 1 above)
Comment (williamFalcon) [reply README.md]: @Andrei-Aksionov if you can't deploy models with litGPT then this isn't useful to enterprises... litGPT is more than just a research tool ;)

Comment (Andrei-Aksionov) [reply README.md]: @williamFalcon It wasn't about LitGPT, more like "Vietnam flashbacks" from one of the projects from my previous job.

**Pull request merged**